v0.3#015 Added log tracing for loki and grafana Added db connection and MONGO_URI Added db user struct Added global Uuid type

UnkoR00T opened the pull request.

23czap commented on src/components/MidGameGoals.vue:
coś zbyt pusto tutaj

23czap commented on src/stores/mainDataStore.ts:
zbyt pusto dodaj pare komentarzy

UnkoR00T committed:
v0.3#015
Added log tracing for loki and grafana
Added db connection and MONGO_URI
Added db user struct
Added global Uuid type
